app/views/dossier/reports/multi.html.slim

```diff
@@ -0,0 +1,9 @@
+div id="#{report.dom_id}"
```

Comment:
Neat trick for future reference. You can call the variable without string interpolation if you like.

div id=report.dom_id